lib/x11utils.pm

```diff
@@ -229,12 +229,7 @@ sub handle_additional_polkit_windows {
         # for S390x testing, since they are not using qemu built-in vnc, it is
         # expected that polkit authentication window can open for first time login.
         # see bsc#1177446 for more information.
-        # Base latest feedback of bsc#1192992,authentication should never open if is_sle  >= 15SP4
-        if (is_sle('>=15-sp4')) {
-            record_soft_failure 'bsc#1192992 - authentication should never open if is_sle >= 15SP4';
-        } else {
-            record_info('authentication open for first time login');
-        }
+        record_info('authentication open for first time login');
```

Comment:
please add the bug reference in the message, it is a strange behavior, better overexplain it.

Reply:
changed && thank you.